.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+# .env
```

Comment:
remove the .env comment

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix